homeassistant/components/circuit/__init__.py

```diff
@@ -0,0 +1,53 @@
+"""The Unify Circuit component."""
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_NAME, CONF_URL
+from homeassistant.helpers import config_validation as cv, discovery
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "circuit"
+CONF_WEBHOOK = "webhook"
+
+_WEBHOOK_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_NAME): cv.string,
+        vol.Required(CONF_URL): vol.All(cv.ensure_list, [cv.string]),
+    }
+)
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.All(
+            cv.ensure_list,
+            [
+                vol.Schema(
+                    {
+                        vol.Optional(CONF_WEBHOOK): vol.All(
+                            cv.ensure_list, [_WEBHOOK_SCHEMA]
+                        ),
+                    }
+                )
+            ],
+        )
+    },
+    extra=vol.ALLOW_EXTRA,
+)
+
+
+async def async_setup(hass, config):
+    """Set up the Unify Circuit component."""
+    domain_config = config.get(DOMAIN, [])
+
+    for conf in domain_config:
+        for webhook_conf in conf.get(CONF_WEBHOOK, []):
```

Comment:
The default empty list can be moved to the config schema.

Reply:
Ok, I can change this, will be added in new commit to PR.